botc/commands/info/claim.py

```diff
@@ -27,7 +28,7 @@
     r"^i(m|'m| am| was) an [^,.]+(\.|,|$)": r"(?<= an )[^,.]+(?=\.|,|$)",
     r"^i(m|'m| am| was) a [^,.]+(\.|,|$)": r"(?<= a )[^,.]+(?=\.|,|$)",
 
-    r"^[^,.]+ here(\.|,|$)": r"[^,.]+(?= here)",
+    r"^[^,.]+(?<!im)(?<!i'm) here(\.|,|$)": r"[^,.]+(?= here)",
```

Comment:
What about `i here`? Not like that's correct grammar but still.

Reply:
Added as an exception